docs/api/throttling.md

```diff
@@ -0,0 +1,33 @@
+# NuGet API rate limiting
```

Comment:
Like @kraigb said, you'll need to add YAML metadata here.  Here's a template.  Make sure you include the opening/closing `---`.

You'll also want to [link your GitHub account to your Microsoft account](https://docs.opensource.microsoft.com/github/linking.html).

```
---
<!-- All articles need the metadata header along with the required values for reporting.</br>
Detailed instructions for completing this template are available in the </br>
Contributor Guide:https://review.docs.microsoft.com/en-us/help/contribute/mock-template-tutorial?branch=master -->
# Mandatory fields. See more on aka.ms/skyeye/meta.
title: Intent and product brand in a unique string of 43-59 chars including spaces - do not include site identifier (it is auto-generated.)
description: 115-145 characters including spaces. Edit the intro para describing article intent to fit here. This abstract displays in the search result.
services: service-name-with-dashes-AZURE-ONLY
keywords: Don’t add or edit keywords without consulting your SEO champ.
author: github-alias
ms.author: MSFT-alias-person-or-DL
ms.date: 12/05/2017
ms.topic: tutorial
# Use only one of the following. Use ms.service for services, ms.prod for on-prem. Remove the # before the relevant field.
# ms.service: service-name-from-white-list
# product-name-from-white-list

# Optional fields. Don't forget to remove # if you need a field.
# ms.custom: can-be-multiple-comma-separated
# ms.devlang:devlang-from-white-list
# ms.suite: 
# ms.tgt_pltfrm:
# ms.reviewer:
# manager: MSFT-alias-manager-or-PM-counterpart
---
```